dp/cloud/go/services/dp/logs_pusher/logs_pusher.go

```diff
@@ -0,0 +1,34 @@
+package logs_pusher
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+)
+
+type DPLog struct {
+	EventTimestamp   int64  `json:"event_timestamp"`
+	LogFrom          string `json:"log_from"`
+	LogTo            string `json:"log_to"`
+	LogName          string `json:"log_name"`
+	LogMessage       string `json:"log_message"`
+	CbsdSerialNumber string `json:"cbsd_serial_number"`
+	NetworkId        string `json:"network_id"`
+}
+
+func PushDPLog(ctx context.Context, log *DPLog, consumerUrl string) error {
+	client := http.Client{}
+	body, err := json.Marshal(*log)
```

Comment:
You don't need to dereference log and a also check for error, since you are marshaling known type.